packages/superset-ui-chart-controls/test/utils/columnChoices.test.tsx

```diff
@@ -40,6 +41,10 @@ describe('columnChoices()', () => {
             verbose_name: 'bar',
           },
         ],
+        verbose_map: {},
+        column_format: { fiz: 'NUMERIC', about: 'STRING', foo: 'DATE' },
+        datasource_name: 'my_datasource',
+        description: 'this is my datasource',
```

Comment:
Bycatch: not sure why linter wasn't picking this up, but the object was missing required props